app/constants.js

```diff
@@ -1,10 +1,12 @@
 const path = require('path')
 
-const { app } = require('electron').remote
+const app = require('electron').app
+const remote = require('electron').remote
+const api = app || remote.app
```

Comment:
I was noticing the unused `requires` myself, then to be sure I didn't miss any, I ran `eshint` (with `standard` as the preset) on just `datasources.js` and I was seeing `Arrow function should not return assignment` errors, so I tweaked those multiline functions.

I could take a side trip and add `eshint` to the project and see how much has to change, with the goal of making it automated. 

My next "i could add" would be about unit testing...

so I updated those.